packages/core/admin/server/register.js

```diff
@@ -14,4 +16,11 @@ module.exports = ({ strapi }) => {
   if (strapi.config.serveAdminPanel) {
     registerAdminPanelRoute({ strapi });
   }
+
+  if (
+    process.env.STRAPI_EXPERIMENTAL === 'true' &&
+    process.env.STRAPI_DISABLE_REMOTE_DATA_TRANSFER !== 'true'
+  ) {
+    registerDataTransferRoute(strapi);
```

Comment:
@Convly @marcoautiero Should we delete the transfer feature entirely for the release, and add it back for 4.7-alpha?